src/index.ts

```diff
@@ -92,9 +96,20 @@ async function main() {
     client_defined = true;
   }
 
+  if (collaboratorBaseUrl && collaboratorUsername && collaboratorLoginTicket) {
+    server.addClient(
+      new CollaboratorClient(
+        collaboratorBaseUrl,
+        collaboratorUsername,
+        collaboratorLoginTicket,
+      ),
+    );
+    client_defined = true;
+  }
+
   if (!client_defined) {
     console.error(
-      "Please set one of REFLECT_API_TOKEN, BUGSNAG_AUTH_TOKEN, API_HUB_API_KEY, QMETRY_API_KEY, ZEPHYR_API_TOKEN, or PACT_BROKER_BASE_URL / (and relevant Pact auth) environment variables",
+      "Please set one of REFLECT_API_TOKEN, BUGSNAG_AUTH_TOKEN, API_HUB_API_KEY, QMETRY_API_KEY, ZEPHYR_API_TOKEN, COLLABORATOR_BASE_URL(and relevant collab environment variables) or PACT_BROKER_BASE_URL / (and relevant Pact auth) environment variables",
```

Comment:
I think this error message is now stretched beyond its initial purpose. I suggest we just replace with:

```suggestion
      "No SmartBear products were configured. Please provide at least one of the required configuration options.",
```